src/components/EmptyState/index.js

```diff
@@ -49,11 +50,13 @@ export class EmptyState extends React.Component {
                 'Morbi malesuada nulla nec purus convallis consequat. Vivamus id mollis quam. Morbi ac commodo nulla. In condimentum orci id nisl volutpat bibendum. Quisque commodo hendrerit lorem quis egestas. Maecenas quis tortor arcu. Vivamus rutrum nunc non neque consectetur quis placerat neque lobortis.'}
             </p>
           </StackItem>
+          {footer ? 
+          footer() :
```

Comment:
Since the only valid child of a `<Stack>` is a `<StackItem>`, we should wrap the `footer()` function in a `<StackItem>`.